hashmap/hashmap.mbt

```diff
@@ -50,6 +51,9 @@ pub fn HashMap::new[K, V](
   ~capacity : Int = default_init_capacity,
   ~hasher : Option[(K) -> Int] = None
 ) -> HashMap[K, V] {
+  if capacity != 0 && not(is_power_of_two(capacity)) {
+    abort("The capacity is not a zero or a power of 2.")
```

Comment:
I think it's too much to abort directly. You can either
- do not expose capacity at all
- resize the capacity to a power of 2 that is larger than the input (which may have problem with input such as 2^31 - 1)

Reply:
Due to the lack of unsigned type and some util functions (like Rust 's `checked_next_power_of_two`). Maybe not exposing the capacity is better.